python/semantic_kernel/connectors/memory/azure_cosmosdb/azure_cosmos_db_memory_store.py

```diff
@@ -61,6 +57,8 @@ def __init__(
 
     @staticmethod
     async def create(
+        cosmos_connstr,
```

Comment:
- The code now incorporates both approaches: providing the conn_string as a variable value and loading it from environment variables if not present.
 - This will ensure consistency with the methodology used in other memory stores within the semantic kernel repo.
- Implementing these changes should resolve the issue with the failing Python document pipeline and enable progress with the PR.